docs/joss_paper/paper.md

```diff
@@ -0,0 +1,156 @@
+This is particularly useful when working with multiple grids along the same dimension; or when assigning different memory layouts to different fields.
+
+In contrast to GYSELA, Gyselalib++ has been conceived as a library, similar to the SeLaLib Fortran library [@selalib], whose independent elements are each unit-tested and can be combined to build a final simulation.
+This design makes the library more versatile, since its elements are not tied to a specific simulation and can be adapted to different needs.
+The shared elements also provide more confidence in the reliability of the implementation as they can prove their validity across multiple applications.
+
+Gyselalib++ includes a range of reusable mathematical operators for plasma simulations.
+These include, but are not limited to, semi-Lagrangian advection schemes, numerical quadrature, differential operators (e.g. finite difference methods), and solvers for common PDEs.
+A complete list of operators is available in the documentation[^doc].
+Many of these tools are designed to work on a variety of grids, including non-uniform grids, which are especially important in edge-region simulations.
+The library also supports MPI-based parallelism, either with distributed operators or with transpositions between different multi-rank storage layouts.
+
+[^doc]: <https://gyselax.github.io/gyselalibxx/>
+
+The VOICE code [@bourne2023] has already been rewritten in C++ using the mathematical tools provided by Gyselalib++.
+Several common simulations including Landau damping (in 2D or 4D Cartesian phase-space coordinates), a bump-on-tail instability (in 2D Cartesian phase-space coordinates), and a guiding-centre model (on polar coordinates) have also been implemented.
+While these examples are included primarily for illustration, they also serve as valuable test-beds for developing and validating new numerical methods.
+As such, these examples are ideal for mathematicians looking to validate new methods in realistic, publication-ready test cases.
+
+## Acknowledgements
+
+This project has received funding from the EoCoE-III project (Energy-oriented Centre of Excellence for Exascale HPC applications).
+This work has been carried out within the framework of the EUROfusion Consortium, funded by the European Union via the Euratom Research and Training Program (Grant Agreement No. 101052200 — EUROfusion).
+Views and opinions expressed are those of the author(s) only and do not necessarily reflect those of the European Union or the European Commission.
+Neither the European Union nor the European Commission can be held responsible for them.
+Emily Bourne’s salary was paid for by the EUROfusion Advanced Computing Hub (ACH).
+This project was provided with computer and storage resources by GENCI CINES thanks to the grant 2024-A0160502224 on the supercomputer  on the supercomputer Adastra’s the GENOA partition.
```

Comment:
Shall we also acknowledge the PTCs ?

> We acknowledge the financial support of the Cross-Disciplinary Program on "Numerical simulation" of CEA, the French Alternative Energies and Atomic Energy Commission.

Reply:
[1ce89e7](https://github.com/gyselax/gyselalibxx/pull/265/commits/1ce89e78621f1b66ed1e347c1ba89d9ed119f21b)